lexers/i/idl.go

```diff
@@ -0,0 +1,34 @@
+package i
+
+import (
+	"strings"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// IDL lexer.
+var Idl = internal.Register(MustNewLexer(
```

Comment:
Let's call it `IDL`. Chroma also uses upper case e.g. for `SQL` (https://github.com/alecthomas/chroma/blob/b44683eae8971945179737e202cfa19c9c7a8dd7/lexers/s/sql.go#L9)
```suggestion
var IDL = internal.Register(MustNewLexer(
```

Reply:
done!